packages/contracts-bedrock/src/cannon/MIPS.sol

```diff
@@ -45,8 +45,8 @@ contract MIPS is ISemver {
     }
 
     /// @notice The semantic version of the MIPS contract.
-    /// @custom:semver 1.1.1-beta.4
-    string public constant version = "1.1.1-beta.4";
+    /// @custom:semver 1.1.1-beta.5
+    string public constant version = "1.1.1-beta.5";
```

Comment:
We should bump the minor version of MIPS.sol as this is a significant change.

For MIPS2.sol (MT-Cannon), it can be just a prerelease bump since it's not deployed anywhere. Though we should bump its minor version as well for consistency.